### PR TITLE
📊 urbanization: GHSL degree of urbanization update

### PR DESCRIPTION
**Pull request by veronikasamborska1994**

### CODE REVIEW

File: etl/steps/data/garden/urbanization/2024-10-14/ghsl_degree_of_urbanisation.meta.yml [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Urbanization
+    processing_level: major
```
Comment (paarriagadap) [inline etl/steps/data/garden/urbanization/2024-10-14/ghsl_degree_of_urbanisation.meta.yml:7]: You can describe a little bit the processing done in `description_processing`

File: etl/steps/data/garden/urbanization/2024-10-14/ghsl_degree_of_urbanisation.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    tb["urban_cluster_popshare"] = tb["urban_cluster_population"] / tb["total_population"] * 100
+    tb["urban_centre_popshare"] = tb["urban_centre_population"] / tb["total_population"] * 100
+
+    tb["urban_cluster_share"] = tb["urban_cluster_area"] / tb["total_area"] * 100
+    tb["urban_centre_share"] = tb["urban_centre_area"] / tb["total_area"] * 100
+
+    tb = tb.drop(columns=["total_population", "total_area"])
+
+    # Create two new dataframes to separate data into estimates and projections (pre-2025 and post-2025 (five year intervals)))
+    past_estimates = tb[tb["year"] < 2025].copy()
```
Comment (paarriagadap) [inline etl/steps/data/garden/urbanization/2024-10-14/ghsl_degree_of_urbanisation.py:93]: Just as a visual preference, you can maybe keep 2020 with data for projections, so the dotted line in [this chart](http://staging-site-urbanization-oct/grapher/urban-share-european-commission) can start where the estimated finished

**Review by paarriagadap — APPROVED** [review]
Good! I left minor comments below.

Also, I would use United Nations as a label in [this line chart](http://staging-site-urbanization-oct/grapher/share-of-urban-population-by-source), instead of "UN"

File: snapshots/urbanization/2024-10-14/ghsl_degree_of_urbanisation.xlsx.dvc [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Global Human Settlement Layer Dataset
+    description: |-
+      This dataset provides a comprehensive classification of urbanization levels for administrative regions worldwide, using the Global ADMinistrative layer version 4.1 (GADM4.1) as the source of administrative units. The classification is derived from the application of the Degree of Urbanisation model's Stage II, which evaluates each unit based on urbanization criteria. It utilizes the Global Human Settlement Layer (GHSL) data, specifically the Settlement Classification (GHS-SMOD_GLOBE_R2023A) that categorizes areas as urban, suburban, or rural, and the Population Grid (GHS-POP_GLOBE_R2023A) that provides detailed population information.
+    date_published: '2024'
```
[the remark this answers is not in the document]
Comment (veronikasamborska1994) [reply snapshots/urbanization/2024-10-14/ghsl_degree_of_urbanisation.xlsx.dvc:9]: actually I have an exact date in the spreadsheet :) thank you!

File: dag/urbanization.yml [hunk 4]
```diff
@@ -64,3 +64,15 @@ steps:
     - data://meadow/un/2024-02-14/sdgs_urbanization
   data://grapher/un/2024-02-14/sdgs_urbanization:
     - data://garden/un/2024-02-14/sdgs_urbanization
+
```
[the remark this answers is not in the document]
Comment (veronikasamborska1994) [reply dag/urbanization.yml:67]: there is still 1 chart that depends on the old version! once the data provider sends me the new one I'll archive it :)

**Pull request merged**